server/odc-service/src/main/java/com/oceanbase/odc/metadb/iam/PermissionRepository.java

```diff
@@ -92,21 +90,26 @@ Optional<PermissionEntity> findByOrganizationIdAndActionAndResourceIdentifier(Lo
     @Query(value = "select p.* from iam_permission p", nativeQuery = true)
     List<PermissionEntity> findAllNoCareExpireTime();
 
+    @Query(value = "select p.* from iam_permission p where p.expire_time <:expireTime",
+            nativeQuery = true)
+    List<PermissionEntity> findByExpireTimeBefore(@Param("expireTime") Date expireTime);
+
     @Modifying
     @Transactional
     @Query(value = "delete from iam_permission p where p.id in (:ids)", nativeQuery = true)
     int deleteByIds(@Param("ids") Collection<Long> ids);
 
     @Modifying
     @Transactional
-    @Query(value = "delete from iam_permission p where p.expire_time is not null and p.expire_time < :expireTime",
-            nativeQuery = true)
-    int deleteByExpireTimeBefore(@Param("expireTime") Date expireTime);
+    @Query(value = "delete from iam_permission p where 1=1", nativeQuery = true)
+    void deleteAll();
```

Comment:
why should we need this method?
`CrudRepository#deleteAll`?

Reply:
Because I add `@Where(clause = "expire_time > now()")` upper `PermissionEntity`. So all not customized query method will auto add this where clause for filtering expired permission. 
In this case, we want to delete all records, so I need to add this customized method.
By the way, this method only used in unit test.